luigi_pipeline/tests/test_seqr_sv_loading.py

```diff
@@ -0,0 +1,303 @@
+        filters=['HIGH_SR_BACKGROUND'], bothsides_support=True, algorithms=['manta'], xpos=1006558902,
+        cpx_intervals=[hl.Struct(type='INV', chrom='1', start=6558902, end=6559723),
+                       hl.Struct(type='DUP', chrom='1', start=6559655, end=6559723)], xstart=1006558902,
+        xstop=1006559723,
+        rg37_locus_end=hl.Locus(contig=1, position=6619783, reference_genome='GRCh37'), svType='CPX',
+        transcriptConsequenceTerms=['INTRONIC', 'CPX'], sv_type_detail='INVdup',
+        geneIds=set({'ENSG00000173662'}), variantId='CPX_chr1_22', sortedTranscriptConsequences=[
+            hl.Struct(gene_symbol='TAS1R1', gene_id='ENSG00000173662', major_consequence='INTRONIC')],
+        docId='CPX_chr1_22',
+        **{"end_locus.contig": "chr1", "end_locus.position": 6559723},
+        # StrVCTVRE_score=0.74,
+    ),
+]
+
+EXPECTED_DATA_GENOTYPES = [
+    hl.Struct(
+        **EXPECTED_DATA_VARIANTS[0],
+        samples_no_call=None, 
+        genotypes=[hl.Struct(sample_id='SAMPLE-1', gq=59, cn=None, num_alt=2),
+                   hl.Struct(sample_id='SAMPLE-2', gq=26, cn=None, num_alt=2),
+                   hl.Struct(sample_id='SAMPLE-3', gq=39, cn=None, num_alt=2),
+                   hl.Struct(sample_id='SAMPLE-4', gq=19, cn=None, num_alt=1),
+                   hl.Struct(sample_id='SAMPLE-5', gq=19, cn=None, num_alt=1)],
+        **{"samples_num_alt.1": {'SAMPLE-4', 'SAMPLE-5'}, "samples_num_alt.2": {'SAMPLE-1', 'SAMPLE-2', 'SAMPLE-3'}},
+        **{key: EXPECTED_SAMPLE_GQ[0].get(key) for key in SAMPLES_GQ_SV_FIELDS}
+    ),
+    hl.Struct(
+        **EXPECTED_DATA_VARIANTS[1],
+        samples_no_call=None,
+        genotypes=[hl.Struct(sample_id='SAMPLE-1', gq=62, cn=None, num_alt=1),
+                   hl.Struct(sample_id='SAMPLE-2', gq=99, cn=None, num_alt=0),
+                   hl.Struct(sample_id='SAMPLE-3', gq=99, cn=None, num_alt=0),
+                   hl.Struct(sample_id='SAMPLE-4', gq=99, cn=None, num_alt=0),
+                   hl.Struct(sample_id='SAMPLE-5', gq=99, cn=None, num_alt=0)],
+                   **{"samples_num_alt.1": {'SAMPLE-1'}, "samples_num_alt.2": None},
+        **{key: EXPECTED_SAMPLE_GQ[1].get(key) for key in SAMPLES_GQ_SV_FIELDS}
+    ),
+    hl.Struct(
+        **EXPECTED_DATA_VARIANTS[2],
+        samples_no_call=None,
+        genotypes=[hl.Struct(sample_id='SAMPLE-1', gq=99, cn=2, num_alt=0),
+                   hl.Struct(sample_id='SAMPLE-2', gq=57, cn=2, num_alt=1),
+                   hl.Struct(sample_id='SAMPLE-3', gq=0, cn=2, num_alt=1),
+                   hl.Struct(sample_id='SAMPLE-4', gq=99, cn=3, num_alt=0),
+                   hl.Struct(sample_id='SAMPLE-5', gq=99, cn=1, num_alt=0)],
+        **{"samples_num_alt.1": {'SAMPLE-2', 'SAMPLE-3'}, "samples_num_alt.2": None},
+        **{key: EXPECTED_SAMPLE_GQ[2].get(key) for key in SAMPLES_GQ_SV_FIELDS}
+    )
+]
+
+
+class SeqrSvLoadingTest(unittest.TestCase):
```

Comment:
Nitpick but maybe 
```suggestion
class SeqrSVLoadingTest(unittest.TestCase):
```
for consistency?